provisioning/service/src/provisioningserviceclient.ts

```diff
@@ -297,6 +299,34 @@ export class ProvisioningServiceClient {
     }, callback);
   }
 
+  /**
+   * Gets the attestation mechanism for an EnrollmentGroup record.
+   * @param enrollementGroupId Unique identifier of the EnrollmentGroup.
+   * @param callback Function called when the request is completed, either with an error or with an AttestationMechanism object.
+   * @returns {Promise<ResultWithHttpResponse<AttestationMechanism>> | void} Promise if no callback function was passed, void otherwise.
+   */
+  public getEnrollmentGroupAttestationMechanism(enrollmentGroupId: string, callback: HttpResponseCallback<AttestationMechanism>): void;
+  public getEnrollmentGroupAttestationMechanism(enrollmentGroupId: string): Promise<ResultWithHttpResponse<AttestationMechanism>>;
+  public getEnrollmentGroupAttestationMechanism(enrollmentGroupId: string, callback?: HttpResponseCallback<AttestationMechanism>): Promise<ResultWithHttpResponse<AttestationMechanism>> | void  {
+    /*Codes_SRS_NODE_PROVISIONING_SERVICE_CLIENT_16_003: [The `getEnrollmentGroupAttestationMechanism` method shall throw a `ReferenceError` if the `enrollementGroupId` parameter is falsy.]*/
+    if (!enrollmentGroupId) {
+      throw new ReferenceError('enrollmentGroupId cannot be \'' + enrollmentGroupId + '\'');
+    }
+
+    /*Codes_SRS_NODE_PROVISIONING_SERVICE_CLIENT_16_004: [The `getEnrollmentGroupAttestationMechanism` shall construct an HTTP request using information supplied by the caller as follows:
+    ```
+    POST /enrollmentgroups/<encodeUriComponentStrict(enrollmentGroupId)>/?api-version=<version> HTTP/1.1
```

Comment:
>(enrollmentGroupId)>/?api-version=<version> [](start = 52, length = 43)

Nit: for this group and for the individual the requirements do NOT include the /attestationmechanism text.  Like I said, just a NIT.